AppController/djinn.rb

```diff
@@ -3121,9 +3121,8 @@ def send_instance_info_to_dashboard()
         http.verify_mode = OpenSSL::SSL::VERIFY_NONE
         response = http.post(url.path, JSON.dump(instance_info),
           {'Content-Type'=>'application/json'})
-        Djinn.log_debug("Done sending instance info to AppDashboard!")
-        Djinn.log_debug("Instance info is: #{instance_info.inspect}")
-        Djinn.log_debug("Response is #{response.body}")
+        Djinn.log_debug("Done sending instance info to AppDashboard: info is: " +
+          "#{instance_info.inspect} response is: #{response.body}")
```

Comment:
Maybe put a delimiter before "response"?
Otherwise the output can be confusing.